docs/getting-started/configure-integration-policy.asciidoc

```diff
@@ -68,6 +68,14 @@ Malware protection also allows you to manage a blocklist to prevent specified ap
 extending the list of processes that {elastic-defend} considers malicious. Use the **Blocklist enabled** toggle
 to enable or disable this feature for all hosts associated with the integration policy. To configure the blocklist, refer to <<blocklist>>.
 
+When *Detect* malware is enabled, {agent} will quarantine any malicious file it finds. Specifically {agent} will remove it from it's current location, obfuscate it (XOR encrypt it with the encryption key `ELASTIC`), move it to a different folder, and rename it as a string like `318e70c2-af9b-4c3a-939d-11410b9a112c`.
+
+- On MacOS, all {agent} versions move quarantined files to the folder `/System/Volumes/Data/.equarantine`.
```

Comment:
Are you sure the MacOS directory is `/System/Volumes/Data/.equarantine` ? It seems to me looking to my installation instance that it is rather directory `/Library/Endgame/.equarantine`

Reply:
@roxana-gheorghe Please confirm

Reply:
The path containing "Endgame" comes from an Endgame sensor, the path for Elastic should be /System/Volumes/Data/.equarantine.